site/src/download/index.hbs

```diff
@@ -18,7 +18,7 @@ layout: default.hbs
       <p>
         If you want to try out OpenLayers without downloading anything (<b>not recommended for production</b>), include the following in the head of your html page:
 <pre><code class="language-html">&lt;script src="https://cdn.jsdelivr.net/gh/openlayers/openlayers.github.io@master/en/{{ version }}/build/ol.js"&gt;&lt;/script&gt;
-&lt;link rel="stylesheet" href="https://cdn.jsdelivr.net/gh/openlayers/openlayers.github.io@master/en/{{ version }}/css/ol.css"&gt;
+&lt;link rel="stylesheet" href="https://cdn.jsdelivr.net/gh/openlayers/openlayers.github.io@master/en/{{ version }}/theme/ol.css"&gt;
```

Comment:
Not quite sure if the changed path will be correct.

Reply:
Yeah, we may revisit this.